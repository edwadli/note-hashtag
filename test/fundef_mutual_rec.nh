
fun First x = (
  Print (StringOfInt x)
  if x > 1 then (Print "s"; Second x; "not") else "done"
)

fun Second x = (
  Print (First (x-1))
)

Print (First 5)
