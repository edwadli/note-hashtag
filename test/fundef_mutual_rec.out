5s4s3s2s1donenotnotnotnot